src/_constants.scss

```diff
@@ -1,3 +1,4 @@
 // durations
-$duration-s: .2s;
-$duration-m: .5s;
+$duration-s: .1s;
+$duration-m: .25s;
+$duration-l: .5s;
```

Comment:
I'd vote on spelling them out for clarity here. Like @mperrotti said, we use `s`,`m`,`l`, for media sizes.

`$duration--short`, `$duration--medium`, `$duration--long`